### ISSUE TITLE
Allow `--uppercase` option

### ISSUE THREAD

**Issue by mhoban** [metabarcoding/obitools4#8]
In the older version of obitools,  most of the tools supported a `--uppercase` option that would output sequences in capital letters (although it didn't always work). This version doesn't seem to support that and just outputs everything in lowercase. It would be nice to have this option again.

**Comment by coissac** [comment]
Yes, you have true, that option disappeared in obitools4. Can you explicit the benefit of such option ?

Thank you

**Comment by mhoban** [comment]
Well, the primary benefit is not having to run a second command to make all the bases uppercase, which can take a long time for large files. It's definitely not critical but it also seems like a fairly simple modification. Especially since most sequence read formats already have bases in uppercase and `obitools 4` is intentionally making them all lowercase.

**Comment by coissac** [comment]
Ok, today I'm focusing on taxonomy and documentation. I keep the idea in mind but with a low priority. But You are welcome to fork and to make a pull request if you think you can achieve the task faster.

Thank you for the suggestion.

**Comment by mhoban** [comment]
I'm not very well-versed in the go language, but I'll take a look. If I think I can implement it, I'll send a PR.

**Comment by mhoban** [comment]
> Yes, you have true, that option disappeared in obitools4. Can you explicit the benefit of such option ?
> 
> Thank you

Just to update this, one area where this is important is when using software like `usearch` that considers lowercase bases to have been masked, and ignores them. In that case, the sequences need to be uppercase to be considered when clustering or denoising.
